falco/client.py

```diff
@@ -2,23 +2,38 @@
 
 from falco.client_credentials import get_grpc_channel_credentials
 from falco.domain import Response
-from falco.svc.output_pb2_grpc import serviceStub
+from falco.svc.outputs_pb2_grpc import serviceStub
 
 
 class InvalidFormat(Exception):
     pass
 
+class TLSConfigError(Exception):
+    print(Exception)
+
 
 class Client:
-    def __init__(self, endpoint, client_crt, client_key, ca_root, output_format=None, *args, **kw):
-        self._client = serviceStub(
-            grpc.secure_channel(
-                endpoint,
-                credentials=get_grpc_channel_credentials(client_crt, client_key, ca_root),
-                options=[("grpc.max_receive_message_length", 1024 * 1024 * 512)],
-            ),
-        )
-        self.output_format = output_format
+    def __init__(self, endpoint, client_crt=None, client_key=None, ca_root=None, output_format=None, *args, **kw):
+        if endpoint.startswith("unix:///"):
```

Comment:
how about creating the channel in the if..else block and leaving out this part?

```
self._client = serviceStub(channel)
self.output_format = output_format
```

Reply:
Good thinking will do